Assets/MixedRealityToolkit-Preview/LightEstimation-Examples/Models/Bunny.obj

```diff
@@ -0,0 +1,4562 @@
+# Blender v2.79 (sub 0) OBJ File: 'BakedBunny.blend'
+# www.blender.org
+mtllib BunnyLow2.mtl
```

Comment:
I don't know how this skipped my mind! Looks like it has no standardized license, just some html about attribution and non-commercial use. Probably easiest to find something else, will work on this.

Reply:
Did we get clarification on the license for this? non-commercial use would likely be a non-starter.

Reply:
The license term looks a bit ambiguous: http://www.graphics.stanford.edu/data/3Dscanrep/ You could use the "Shaderball" model that already exists in the MRTK since it was created by Microsoft.

Reply:
> You could use the "Shaderball" model that already exists in the MRTK

I like this idea!

Reply:
Yep! Switched to the ShaderBall. It looks pretty great! :)